deployment/cfn/stacks.py

```diff
@@ -80,3 +87,19 @@ def build_stacks(mmw_config, aws_profile, **kwargs):
         tiler_graph.go()
         application_graph.go()
         worker_graph.go()
+
+    if kwargs['activate_dns']:
+        public_hosted_zone_graph.go()
+
+
+def destroy_stacks(mmw_config, aws_profile, **kwargs):
+    """Destroy stacks that are associated with stack_color"""
+    region = mmw_config['Region']
+    stack_color = kwargs['stack_color']
+
+    cfn_conn = cfn.connect_to_region(region, profile_name=aws_profile)
+    stacks = reversed(sorted(cfn_conn.describe_stacks(),
```

Comment:
Forgot to take this out. I was using it to try and reproduce the race conditions. Removed in https://github.com/WikiWatershed/model-my-watershed/commit/33578be.